### PR TITLE
Rbotha/feature 67 week 6 view map hazard detail

### PR DESCRIPTION
**Pull request by ReinhardtBotha**
Hazard Detail Card:
- HazardDetailCard.js component to contain all the hazard details.
- hazard-detail-card.css for additional styling

Added the functionality in the home script.js to open and populate the hazard detail card when clicking on a hazard marker on the map.

### CODE REVIEW

File: pages/home/script.js [hunk 1]
```diff
@@ -326,5 +362,32 @@ const onSearchInput = debounce(async ({ target }) => {
...
-
 });
+
+// Load Hazard Detail Card
+let hazardReportID = '16cde280-ac58-467b-888e-dd0549274b6e';
+let currentReport = {};
+const body = document.getElementById('home-body');
+
+// Calculate distance from user to hazard with Haversine foruma
+function calcHazardDistance(lat1, lon1, lat2, lon2) {
```
Comment (BryanCaldeira) [inline pages/home/script.js:373]: Can mostly be a helper function. Fine with it for now... I'll do the code polishing later if that's fine

Comment (ReinhardtBotha) [reply pages/home/script.js:373]: Ahh... good idea! sure we can look at that later.

File: assets/css/hazard-detail-card.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  align-items: center;
+  gap: 0.5rem;
+  align-self: stretch;
+}
+
+#user-image {
+  width: 1.75rem;
+  height: 1.75rem;
+  border-radius: 6.25rem;
+}
```
Comment (ALFAROSO) [inline assets/css/hazard-detail-card.css:51]: #report-card__picture-container {
  overflow: hidden;
}

Comment (ReinhardtBotha) [reply assets/css/hazard-detail-card.css:54]: Thanks! I used `overflow: scroll;` to match the designs, but also added some width constrains to make it function correctly.

**Review by whamester — APPROVED** [review]

**Pull request merged**